tests/integration/test_storage_s3/test.py

```diff
@@ -277,9 +281,9 @@ def test_put_get_with_globs(cluster):
 
 # Test multipart put.
 @pytest.mark.parametrize("maybe_auth,positive", [
-    ("", True),
+    ("", True)
     # ("'minio','minio123',",True), Redirect with credentials not working with nginx.
-    ("'wrongid','wrongkey',", False)
+    # ("'wrongid','wrongkey',", False) ClickHouse crashes in some time after this test, local integration tests run fails.
```

Comment:
Why this happens? Did it happened before changes?

Reply:
Yes, the problem reproduces on the master branch.